api/routes/apiRoutes.js

```diff
@@ -4,121 +4,137 @@ module.exports = function(app) {
     * @group state - Operations about system states
-    * @param {String} uvc_status - ['On', 'Off']  Default: 'Off '
-    * @param {Number} uvc_most_recent - Logs most recent use of UVC
-    * @param {String} door_status - ['Open', 'Closed'] Default: 'Open' 
-    * @param {String} mask_status - ['Mask Present', 'No Mask Present']  Default: 'No Mask Present'
+    * @param {String} keyword - ['uvc', 'door', 'mask']
+    * @returns {object} 204 - Deleted
+    * @returns {Error}  404 - Bad Request: Cannot register status.
+    */
+
+    app.route('/state/:statusId')
+        .delete(state.delete_status);
+
+    /**
+    * This function comment is parsed by doctrine
+    * @route POST /userHome/user-home
+    * @group userHome - Operations about user home status 
     * @param {String} user_status - ['User Home', 'User Not Home']
-    * @returns {object} 200 - OK
+    * @param {Date} status_time - Time at which user status is logged. 
+    * @returns {object} 201 - Created 
     * @returns {Error}  default - Unexpected error
     */
-
+    
+    app.route('/userHome/user-status')
+        .post(userHome.set_user_home);  
+    
     /**
     * This function comment is parsed by doctrine
-    * @route PUT /state/door-switch
+    * @route GET /userHome/user-home
     * @group state - Operations about system states
-    * @param {String} uvc_status - ['On', 'Off']  Default: 'Off '
-    * @param {Number} uvc_most_recent - Logs most recent use of UVC
-    * @param {String} door_status - ['Open', 'Closed'] Default: 'Open' 
-    * @param {String} mask_status - ['Mask Present', 'No Mask Present']  Default: 'No Mask Present'
     * @param {String} user_status - ['User Home', 'User Not Home']
+    * @param {Date} status_time - Time at which user status is logged. 
     * @returns {object} 200 - OK
     * @returns {Error}  default - Unexpected error
     */
 
-    app.route('/state/door-switch')
-        .get(state.read_switch_open_close)
-        .put(state.update_switch_open_close);
-
     /**
     * This function comment is parsed by doctrine
-    * @route PUT /state/user-home
+    * @route PUT /userHome/user-home
     * @group state - Operations about system states
-    * @param {String} uvc_status - ['On', 'Off']  Default: 'Off '
-    * @param {Number} uvc_most_recent - Logs most recent use of UVC
-    * @param {String} door_status - ['Open', 'Closed'] Default: 'Open' 
-    * @param {String} mask_status - ['Mask Present', 'No Mask Present']  Default: 'No Mask Present'
     * @param {String} user_status - ['User Home', 'User Not Home']
+    * @param {Date} status_time - Time at which user status is logged. 
     * @returns {object} 200 - OK
     * @returns {Error}  default - Unexpected error
     */
-
-    app.route('/state/user-home')
-        .put(state.update_user_home);
     
+    /**
+    * This function comment is parsed by doctrine
+    * @route DELETE /userHome/user-home
+    * @group state - Operations about system states
```

Comment:
No mention of `:userHomeId` in docs

Reply:
Good catch. This has been resolved in the most recent push.